Various grammar and format tweaks

cherylhughey opened the pull request:
- [X] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Various grammar and format tweaks

kingthorin commented on Testing_for_APIs.md:
I'd suggest not putting too much time into this one. It obviously needs a bunch of work.

kingthorin replied:
That's likely why it's sitting in the root and not yet placed.

kingthorin commented on checklist/Testing_Checklist.md:
```diff
@@ -37,7 +37,7 @@ Note: The `Status` column can be set for values similar to "Pass", "Fail", "N/A"
 | WSTG-IDNT-05      | Testing for Weak or Unenforced Username Policy                             |        |       |
 | **WSTG-ATHN**     | **Authentication Testing**                                                 |        |       |
-| WSTG-ATHN-01      | Testing for Credentials Transported over an Encrypted Channel              |        |       |
+| WSTG-ATHN-01      | Testing for Credentials Transported Over an Encrypted Channel              |        |       |
```
This was already correct for the Chicago style of titles that the project uses.
https://capitalizemytitle.com/#Chicago

kingthorin commented on document/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/12-Test_for_Content_Security_Policy.md:
```diff
@@ -16,7 +16,7 @@ Content Security Policy supports directives which allow granular control to the
 ## How to Test
 
-To test for misconfigurations in CSPs look for insecure configurations by examining `Content-Security-Policy` HTTP response header or CSP `meta` element in a proxy tool:
+To test for misconfigurations in CSPs, look for insecure configurations by examining `Content-Security-Policy` HTTP response header or CSP `meta` element in a proxy tool:
```
```suggestion
To test for misconfigurations in CSPs, look for insecure configurations by examining the `Content-Security-Policy` HTTP response header or CSP `meta` element in a proxy tool:
```

kingthorin commented on template/999-Foo_Testing/1-Testing_for_a_Cat_in_a_Box.md:
```diff
@@ -57,7 +57,7 @@ If no cat reveals itself, it is unlikely there is a cat in the box. However, thi
 ### Use a Box Camera
 
-Set up a wireless camera inside the box to continuously monitor for cats inside. This test requires the following steps.
+Set up a wireless camera inside the box to continuously monitor for cats inside. This test requires the following steps:
```
This was good as-is.

cherylhughey commented:
Proof reading changes.

kingthorin commented:
Please also add yourself to: https://github.com/OWASP/wstg/tree/master/document/1-Frontispiece#reviewers-or-editors

kingthorin commented:
Did you close this on purpose?

cherylhughey commented:
Sorry, I thought I was supposed to close it when done.

kingthorin commented:
No the items from the review need to be addressed then we’ll merge the changes.

kingthorin commented:
@cherylhughey so if you could just push additional changes addressing the review comments we will get this merged.

cherylhughey commented:
I took a look and have no further changes to push. Thank you.

________________________________
From: Rick M ***@***.***>
Sent: Friday, June 4, 2021 6:08 AM
To: OWASP/wstg ***@***.***>
Cc: Cheryl Hughey ***@***.***>; Mention ***@***.***>
Subject: Re: [OWASP/wstg] Various grammar and format tweaks (#757)


@cherylhughey<https://github.com/cherylhughey> so if you could just push additional changes addressing the review comments we will get this merged.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/OWASP/wstg/pull/757#issuecomment-854621491>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AGYL4UPYIJQ6YLB24XYBG23TRCXZZANCNFSM457ZJSYQ>.

kingthorin commented:
Thanks
